specification/redisenterprise/resource-manager/Microsoft.Cache/preview/2020-10-01-preview/redisenterprise.json

```diff
@@ -0,0 +1,1664 @@
+      "properties": {
+        "minimumTlsVersion": {
+          "type": "string",
+          "description": "The minimum TLS version for the cluster to support, e.g. '1.2'"
+        },
+        "hostName": {
+          "type": "string",
+          "readOnly": true,
+          "description": "DNS name of the cluster endpoint"
+        },
+        "provisioningState": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Current status of the cluster",
+          "enum": [
+            "Succeeded",
+            "Failed",
+            "Creating",
+            "Updating",
+            "Deleting",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "redisVersion": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Version of redis the cluster supports, e.g. '6'"
+        },
+        "privateEndpointConnections": {
+          "type": "array",
+          "readOnly": true,
+          "description": "List of private endpoint connections associated with the specified RedisEnterprise cluster",
+          "items": {
+            "$ref": "../../../../../common-types/resource-management/v1/privatelinks.json#/definitions/PrivateEndpointConnection"
+          }
+        }
+      }
+    },
+    "ClusterList": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Cluster"
+          },
+          "description": "List of clusters."
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The URI to fetch the next page of results."
+        }
+      },
+      "description": "The response of a list-all operation."
+    },
+    "Database": {
+      "description": "Describes a database on the RedisEnterprise cluster",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/DatabaseProperties",
+          "description": "Other properties of the database."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ]
+    },
+    "DatabaseUpdate": {
+      "description": "A partial update to the RedisEnterprise database",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/DatabaseProperties",
+          "description": "Properties of the database."
+        }
+      }
+    },
+    "DatabaseProperties": {
+      "title": "RedisEnterprise database properties",
+      "description": "Properties of RedisEnterprise databases, as opposed to general resource properties like location, tags",
+      "properties": {
+        "clientProtocol": {
```

Comment:
We also need 'clusteringPolicy' property which is enum with values 'EnterpriseCluster' (non-cluster-mode emulation),  or 'OSSCluster' (OS redis cluster mode)

Reply:
is the 'default' database that comes by default using EnterpriseCluster or OSSCluster?

Reply:
OSSCluster is the default when not specified